### DISCUSSION TITLE
Static caching for the REST API

### DISCUSSION THREAD

**Discussion by LorensoD** [statamic/cms#9786] [Q&A]
I'm trying to understand if it's possible to use [static caching](https://statamic.dev/static-caching#important-preface) for the [REST API](https://statamic.dev/rest-api#caching) endpoints. Has anyone been able to do this? Or is it currently not supported in Statamic?

The default REST API caching is off course very fast. Static caching would be even better since it would decrease my server load and increase my Next.js page generation times. ⚡️

**Comment by duncanmcclean** [comment]
Static Caching only works on the front-end, not for the REST or GraphQL APIs.

Like you mentioned, the REST API has [its own caching system](https://statamic.dev/rest-api#caching) instead.

**Reply by jasonvarga** [reply]
The default caching system for the API is basically half-measure static caching.
